var/spack/repos/builtin/packages/singularity/package.py

```diff
@@ -24,8 +24,11 @@ class Singularity(MakefilePackage):
     url      = "https://github.com/sylabs/singularity/releases/download/v3.1.1/singularity-3.1.1.tar.gz"
     git      = "https://github.com/sylabs/singularity.git"
 
+    maintainers = ['ArangoGutierrez','alalazo']
```

Comment:
I think flake8 will complain and say you need an extra space here:
```suggestion
    maintainers = ['ArangoGutierrez', 'alalazo']
```